pkg/manifests/manifests.go

```diff
@@ -2294,7 +2294,9 @@ func (f *Factory) ControlPlanePrometheusRule() (*monv1.PrometheusRule, error) {
 			switch g.Name {
 			case "kubernetes-system-apiserver",
 				"kubernetes-system-controller-manager",
-				"kubernetes-system-scheduler":
+				"kubernetes-system-scheduler",
+				"kubernetes-apiserver-slos",
+				"kubernetes-apiserver.rules":
```

Comment:
Since these rules were removed from the manifest, there is no need to skip them since they don't exist anymore.
I'm pretty sure it is the same for the two others.

Reply:
good catch! :+1:

Reply:
got it! will fix. yeah, i wasn't sure whether changing this file is necessary.

Reply:
to confirm, the changes in this file isn't necessary, but `jsonnet/patch-rules.libsonnet` still needs to be updated, right?

Reply:
yes, the changes in `jsonnet/patch-rules.libsonnet` will allow you to prevent the rules from being generated in the manifests. Meanwhile the changes here were only applied when reading the manifest. Since you updated it in a way that it removed the rules, they won't be seen at this level anymore.